### PR TITLE
Build aarch64

### PR DESCRIPTION
**Pull request by unkcpz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by unkcpz** [comment]
@conda-forge-admin, please rerender

**Comment by unkcpz** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -37,6 +37,9 @@ export OMPI_MCA_plm_rsh_agent=sh
 # Only pw, cp, and unit tests are safe to run when using cmake curently (to fix in later releases)
 #make test
 # there are known test failures that will be addressed later
+# disable test for aarch64 for now it takes too long and timeout of build
+if [[ $(lscpu | awk '/Architecture:/{print $2}') != "aarch64" ]]; then 
 ctest -L "pw|cp|unit" -LE epw --output-on-failure  || true
```
Comment (ltalirz) [inline recipe/build.sh:42]: Can you run at least _some_ of the tests?

I don't know which ones make most sense, but "unit" sounds like they are fast

**Comment by unkcpz** [comment]
@conda-forge-admin, please rerender

**Comment by unkcpz** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh (hunk 1 above)
Comment (unkcpz) [reply recipe/build.sh]: image

I think the 60s more should be enough. I'll try to run it once more, if it still not able to finish all, let's just keep it without tests?

**Comment by unkcpz** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh (hunk 1 above)
Comment (unkcpz) [reply recipe/build.sh]: Good, all tests passed this time. Whereas it is a problem that the next time the test may fail again.

**Pull request merged**